extensions/security-jpa/runtime/src/main/resources/META-INF/quarkus-extension.json

```diff
@@ -0,0 +1,9 @@
+
+{
+    "name": "Quarkus Security JPA Support",
+    "labels": [
+        "security",
+        "jpa"
+    ],
+    "guide": "https://quarkus.io/guides/security-jpa"
```

Comment:
This should be a YAML file now.

Reply:
OK thanks, fixed. Damn this was a long-standing PR :(